core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -10627,4 +10648,15 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         .withSpark()
         .ok(expectedSparkSql);
   }
+
+  @Test public void testSortByOrdinalForSpark() {
+    final String query = "SELECT \"product_id\" from \"product\"\n"
+        + "order by \"product_id\"";
```

Comment:
As this is test case for sort by Ordinal .. plz mention ORDER BY 1, as 1 will here point to 1st column